mirgecom/initializers.py

```diff
@@ -807,3 +808,77 @@ def exact_rhs(self, discr, q, t=0.0):
 
         return join_conserved(dim=self._dim, mass=massrhs, energy=energyrhs,
                               momentum=momrhs, species_mass=yrhs)
+
+
+class MixtureInitializer:
+    r"""Solution initializer for multi-species mixture.
```

Comment:
If this is uniform, would it be possible to generalize the existing `Uniform` initializer?

Reply:
I added some docs that will hopefully help make it more apparent how this init differs from the others.  The main thing is mixture-compatible EOS. 

(6132569)